---
tags:
  - source/journalArticle
  - zotero
doi: "10.1073/pnas.2304663120"
itemKey: QDI9D7SP
---
>[!metadata]+
> Enumerating soil biodiversity
> [[Anthony, Mark A.]], [[Bender, S. Franz]], [[van der Heijden, Marcel G. A.]], 
> [[Proceedings of the National Academy of Sciences (journal)]] (2023)
> important, 
> [Online link](https://www.pnas.org/doi/10.1073/pnas.2304663120), [Zotero Item](zotero://select/library/items/QDI9D7SP),[Local (Full Text)](file://C:/Users/aburg/Documents/references/zotero/storage/XHXDIVK2/Anthony2023_Enumeratingsoil.pdf), 


>[!abstract]-
>Soil is an immense habitat for diverse organisms across the tree of life, but just how many organisms live in soil is surprisingly unknown. Previous efforts to enumerate soil biodiversity consider only certain types of organisms (e.g., animals) or report values for diverse groups without partitioning species that live in soil versus other habitats. Here, we reviewed the biodiversity literature to show that soil is likely home to 59 ± 15% of the species on Earth. We therefore estimate an approximately two times greater soil biodiversity than previous estimates, and we include representatives from the simplest (microbial) to most complex (mammals) organisms. Enchytraeidae have the greatest percentage of species in soil (98.6%), followed by fungi (90%), Plantae (85.5%), and Isoptera (84.2%). Our results demonstrate that soil is the most biodiverse singular habitat. By using this estimate of soil biodiversity, we can more accurately and quantitatively advocate for soil organismal conservation and restoration as a central goal of the Anthropocene.

## notes %% begin notes %%
[[taxonomy]]
> The first main obstacle to synthesizing global biodiversity is understanding the “species” concept across different life forms. Most familiar are the biological and evolutionary species concepts which define species as populations incapable of effectively mating with other groups (13) or as separately evolving lineages with different heritable traits at the population level (14). There are secondary species concepts that better account for all types of organisms using operational methods to distinguish species, including asexually and nonreproducing groups (15). Differences among these concepts are most salient when comparing macrobes to microbes. For example, whereas mammal species diversity is often based on direct observations in the field paired with information on species distributions, traits, and molecular analyses (16), microbial species diversity has been quantified primarily using molecular analyses of DNA hybridization (17), ribosomal RNA gene polymorphism (18), and/or dissimilarity in other regions of the microbial genome (19). Microbial species differences are also blurred by horizontal gene transfer (20) and/or asexual reproduction (21), but operationally, the vast majority of researchers define microbial species based on sequence similarity of ribosomal RNA regions (22–24), enabling a fairly straightforward integration across microbial studies (25). The species concept is even muddier for viruses due to immense ranges in mutation rates, lack of ribosomal and other replication-related protein-encoding genes, and the absence of reproduction (26). Yet, recent work demonstrates that viruses can be organized into isolated groups akin to the biological species concept (26), and overlapping viral RNA and DNA sequences can be used as indicators of taxa much like other microbial ribosomal RNA gene regions (27, 28). However, it is worth noting that the viral species concept is widely debated for myriad reasons (e.g., some viruses lack the ability to recombine genomes, viruses cannot replicate without hosts), and arguably, the viral species concept is most useful for taxonomic purposes (29, 30). Thus, while the species concept may vary considerably across groups, especially between macrobes and various microbes, there are now established methods and sufficient data to synthesize findings using best practices specific to each area of life.
%% end notes %%

%% Import Date: 2025-06-11T14:41:43.127+02:00 %%
