---
tags:
  - source/journalArticle
  - zotero
doi: "10.1016/j.soilbio.2024.109508"
itemKey: XJ7XR94E
importDate: 2025-07-09
---
>[!metadata]+
> Land use intensity is a major driver of soil microbial and carbon cycling across an agricultural landscape
> [[Byers, Alexa K.]], [[Condron, Leo]], [[Wakelin, Steve A.]], [[Black, Amanda]], 
> [[Soil Biology and Biochemistry (journal)]] (2024)
> Land use change, Agricultural landscape, Metagenomic DNA sequencing, Phospholipid fatty acid analysis, Soil carbon, Soil microbiome, 
> [Online link](https://www.sciencedirect.com/science/article/pii/S0038071724001974), [Zotero Item](zotero://select/library/items/XJ7XR94E), [Local (ScienceDirect Full Text PDF)](file://C:/Users/aburg/Documents/references/zotero/storage/Q8FFN6VE/Byers2024_Landuse.pdf), 

>[!abstract]-
>Soil carbon (C) storage is a critical ecosystem function that underpins human health and well-being. The acceleration of human-driven land use change, such as agricultural intensification, is a major driver of soil C loss globally. Developing sustainable land use practices that enhance agricultural productivity whilst protecting essential ecosystem functions such as soil C storage is vital. The soil microbiome has a critical role in regulating soil biogeochemical cycling processes, including soil C cycling. Examining the impacts of land use intensity on the soil microbiome enables us to assess the potential effects on long-term soil C stocks. Using metagenomic DNA sequencing and phospholipid fatty acid analysis, we investigated differences in the activity, diversity, and function of the soil microbiome associated with five contrasting land uses across an agricultural landscape. The land uses covered a gradient of disturbance intensities and included remnant native forest, regenerating native bush, exotic plantation forest, dryland pasture, and irrigated pasture. We identified pronounced differences in the soil microbiome associated with each land use, including the diversity and abundance of microbial C and nitrogen (N) cycling genes. Notably, intensive agricultural land uses had a significantly higher diversity and abundance of microbial C-degrading genes, whilst land uses of remnant native forest had the lowest diversity and abundance of microbial C-degrading genes. Our findings suggest that intensive agricultural land use may increase the functional potential of the soil microbiome to mineralize soil C, potentially resulting in a greater loss of soil C as respired CO2 into the atmosphere. This research may be used to support the development of sustainable management practices that promote the persistence of soil C across agricultural landscapes, such as the protection of remnant native forest fragments and greater incorporation of regenerating native vegetation.

%% begin notes %%
[[environmental DNA#soil sampling]]
> The number of replicate sites (n) of each land use class varied depending on availability and access (n = 3 for native forest and exotic forest; n = 5 for irrigated pasture, dryland pasture, and regenerating bush). The locations of the sites sampled are provided in [Fig. S1](https://www.sciencedirect.com/science/article/pii/S0038071724001974#appsec1), Supplementary File 2. To account for the variation in landscape topography in statistical analyses, the slope angle of each site was extracted from an 8 m [digital elevation model](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/digital-elevation-model "Learn more about digital elevation model from ScienceDirect's AI-generated Topic Pages") (LINZ Data Service, [https://data.linz.govt.nz/](https://data.linz.govt.nz/)). Slope classes were assigned based on slope angle as follows: low slope, 1 to 12°; medium slope, 13 to 25°; high slope, over 25° ([Mackay et al., 2021](https://www.sciencedirect.com/science/article/pii/S0038071724001974#bib64)). [Soil sampling](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/soil-sampling "Learn more about Soil sampling from ScienceDirect's AI-generated Topic Pages") was carried out from June to July 2022. In each site, five sampling points were marked out in the centre and four corners of a 5 m2 grid. At each sampling point, the litter layer was removed, and four soil cores were collected to a 10 cm depth (A horizon) and combined to provide a 500 g composite sample. Five soil samples were collected per site; three were used for chemical and biological analyses and two were stored for future research. Once back in the laboratory, moist soils were sieved to less than 2 mm and subsamples were stored at either −20 °C or 4 °C depending on the subsequent laboratory analysis.

^a45e7f

> Shotgun metagenomic sequencing was performed to enable us to examine differences in diversity, composition, and abundance of microbial functional genes related to soil C and N cycling between the land uses ([Manoharan et al., 2017](https://www.sciencedirect.com/science/article/pii/S0038071724001974#bib68)). [Genomic DNA](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/genomic-dna "Learn more about Genomic DNA from ScienceDirect's AI-generated Topic Pages") was extracted from 250 mg of each soil sample using the DNeasy PowerSoil Pro Kit (Qiagen) following the manufacturer's instructions. DNA quality was assessed using a Nanodrop spectrophotometer (A260:A280 ratio) and quantity was determined on a [Qubit](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/quantum-dot "Learn more about Qubit from ScienceDirect's AI-generated Topic Pages") 2.0 fluorometer. DNA samples were sent for shotgun metagenomics sequencing through the Illumina NovaSeq PE150 platform (NovogeneAIT Genomics, Singapore). Quality summaries for the raw sequencing data are provided in [Table S2](https://www.sciencedirect.com/science/article/pii/S0038071724001974#appsec1), Supplementary File 2. Quality-filtered, paired-end metagenomic reads were mapped against a masked human reference genome using BBMap version 39.01 ([Bushnell, 2014](https://www.sciencedirect.com/science/article/pii/S0038071724001974#bib10)) to remove any potential DNA from human sources.

^cea6c2

[[biodiversity dynamics]]
> Land use intensity was associated with significant shifts in the composition of microbial functional genes

^d9787f


%% end notes %%

%% Import Date: 2025-07-09T11:58:06.101+02:00 %%
