---
tags:
  - source/journalArticle
  - zotero
doi: "10.3389/fmicb.2021.682886"
itemKey: UPTS83JM
---
>[!metadata]+
> Beyond Taxonomic Identification: Integration of Ecological Responses to a Soil Bacterial 16S rRNA Gene Database
> [[Jones, Briony]], [[Goodall, Tim]], [[George, Paul B. L.]], [[Gweon, Hyun S.]], [[Puissant, Jeremy]], [[Read, Daniel S.]], [[Emmett, Bridget A.]], [[Robinson, David A.]], [[Jones, Davey L.]], [[Griffiths, Robert I.]], 
> [[Frontiers in Microbiology (journal)]] (2021)
> 16S Database, Amplicon 16S rRNA, Countryside Survey, Ecological responses, HOF (Huisman, Olff & Fresco) modelling, Soil bacteria communities, Traits, 
> [Online link](https://www.frontiersin.orghttps://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2021.682886/full), [Zotero Item](zotero://select/library/items/UPTS83JM),[Local (Full Text PDF)](file://C:/Users/aburg/Documents/references/zotero/storage/4B5EJ9RQ/Jones2021_TaxonomicIdentification.pdf), 


>[!abstract]-
>High-throughput sequencing 16S rRNA gene surveys have enabled new insights into the diversity of soil bacteria, and furthered understanding of the ecological drivers of abundances across landscapes. However, current analytical approaches are of limited use in formalizing syntheses of the ecological attributes of taxa discovered, because derived taxonomic units are typically unique to individual studies and sequence identification databases only characterize taxonomy. To address this, we used sequences obtained from a large nationwide soil survey (GB Countryside Survey, henceforth CS) to create a comprehensive soil specific 16S reference database, with coupled ecological information derived from survey metadata. Specifically, we modeled taxon responses to soil pH at the OTU level using hierarchical logistic regression (HOF) models, to provide information on both the shape of landscape scale pH-abundance responses, and pH optima (pH at which OTU abundance is maximal). We identify that most of the soil OTUs examined exhibited a non-flat relationship with soil pH. Further, the pH optima could not be generalized by broad taxonomy, highlighting the need for tools and databases synthesizing ecological traits at finer taxonomic resolution. We further demonstrate the utility of the database by testing against geographically dispersed query 16S datasets; evaluating efficacy by quantifying matches, and accuracy in predicting pH responses of query sequences from a separate large soil survey. We found that the CS database provided good coverage of dominant taxa; and that the taxa indicating soil pH in a query dataset corresponded with the pH classifications of top matches in the CS database. Furthermore we were able to predict query dataset community structure, using predicted abundances of dominant taxa based on query soil pH data and the HOF models of matched CS database taxa. The database with associated HOF model outputs is released as an online portal for querying single sequences of interest (https://shiny-apps.ceh.ac.uk/ID-TaxER/), and flat files are made available for use in bioinformatic pipelines. The further development of advanced informatics infrastructures incorporating modeled ecological attributes along with new functional genomic information will likely facilitate large scale exploration and prediction of soil microbial functional biodiversity under current and future environmental change scenarios.

## notes %% begin notes %%

%% end notes %%

%% Import Date: 2025-06-11T14:41:46.613+02:00 %%
