---
tags:
  - source/journalArticle
  - zotero
doi: "10.1038/nrmicro3468"
itemKey: VKAUNQ6C
importDate: 2025-07-09
---
>[!metadata]+
> Dispersing misconceptions and identifying opportunities for the use of 'omics' in soil microbial ecology
> [[Prosser, James I.]], 
> [[Nature Reviews Microbiology (journal)]] (2015)
> Metagenomics, Microbial ecology, Microbiome, Soil microbiology, 
> [Online link](https://www.nature.com/articles/nrmicro3468), [Zotero Item](zotero://select/library/items/VKAUNQ6C), [Local (Full Text PDF)](file://C:/Users/aburg/Documents/references/zotero/storage/T4PH35WQ/Prosser2015_Dispersingmisconceptions.pdf), 

>[!abstract]-
>In this Opinion article, James Prosser considers the conceptual limitations of metagenomics and metatranscriptomics in contributing to our understanding of soil microbial ecology, and also explores potential opportunities for using these techniques to address specific ecological questions.

%% begin notes %%
[[environmental DNA]]

## Quotes
> Understanding the ecology of soil microorganisms (the forces shaping soil microbial communities, their response to environmental change and their physicochemical and biological interactions).

> Molecular studies (those that describe microbial taxonomy on the basis of the 16S rRNA gene) indicate that up to 1 million different bacterial and archaeal species are present in 10 g of soil, in the context of approximately 1 billion microbial cells[3](https://www.nature.com/articles/nrmicro3468#ref-CR3 "Gans, J. Computational improvements reveal great bacterial diversity and high metal toxicity in soil. Science 309, 1387–1390 (2005)."),[4](https://www.nature.com/articles/nrmicro3468#ref-CR4 "Schloss, P. D. & Handelsman, J. Toward a census of bacteria in soil. PLoS Comput. Biol. 2, e92 (2006)."),[5](https://www.nature.com/articles/nrmicro3468#ref-CR5 "Roesch, L. et al. Pyrosequencing enumerates and contrasts soil microbial diversity. ISME J. 1, 283–290 (2007).").

> It is also unlikely that genomes constructed from metagenomes will be derived from a single cell; rather, they will be a chimaera of genomes with undefined similarity.

![[Prosser2015_Dispersingmisconceptions_metagenomics_process.webp]]

![[Prosser2015_Dispersingmisconceptions_genome_gene_centric.webp]]
Gpb: giga pair bases (bilions of pairs of DNA nucleotides)
> complete coverage of a soil metagenome is not usually obtained; for example, in one study, it was shown that deep coverage of the majority of a soil community was not achieved, even with 300 Gbp of sequence data[29](https://www.nature.com/articles/nrmicro3468#ref-CR29 "Howe, A. C. et al. Tackling soil diversity with the assembly of large, complex metagenomes. Proc. Natl Acad. Sci. USA 111, 4904–4909 (2014).").

> the presence of a functional gene is not evidence of its activity. The host organism may be dormant, inactive or only active when alternative metabolic pathways that do not require the function of the detected gene are expressed. The gene may be cryptic, the gene transcript may not be translated, and the prevailing environmental conditions (including substrate concentration, temperature, pH and water availability) may inhibit activity

> Soil microbial communities contain many genes that encode mutually exclusive processes; for example, nitrification and denitrification are favoured in aerobic and anaerobic conditions, respectively, and predicting process rates from the abundances of functional genes associated with each pathway is implausible without information on oxygen concentration and substrate availability. In addition, a single enzyme may participate in several distinct physiological pathways and contribute to different ecosystem functions or may itself have more than one function

> Even if absolute gene abundance is known, there is little evidence that this is a reliable, quantitative measure of physiological activity or ecosystem function as it assumes constant enzyme activity per gene, which is unlikely if the same functional gene (and its encoded enzyme) is present in cells with different properties.

> It is also difficult to draw meaningful information from correlations between the physicochemical characteristics of soil and metagenomic data. For example, temporal heterogeneity invalidates any potential links between the abundance of a functional gene and the environmental characteristics that influence both the associated process and the composition of the community containing this gene, such that the gene may be present in previously active cells that are now dormant.

> However, the metagenome that is present at any point in time may provide historical information on microbial functions, processes or pathways that have operated in the past and on the environmental conditions that existed at that time. That said, we have virtually no basis on which to interpret the historical information provided by such metagenomic information, but consideration of the question alone can generate interesting avenues for research. For example, for how long are genes stable in soil, which soil characteristics enhance the retention and stability of genes and which characteristics lead to the rapid turnover of genes?


%% end notes %%

%% Import Date: 2025-07-09T15:24:31.965+02:00 %%
