---
tags:
  - concept/idea
---
eDNA methods for invertebrates communities is not mature yet, different markers lead to different results, and taxon specific results (annelids vs arthropods), and anyway most OTU remain unidentified ([[Kirse2021_UnearthingPotential]])

there is no clear link between [[metagenomics]] and [[ecosystem function]] ([[Prosser2015_Dispersingmisconceptions]])

the presence and abundance of genes does not mean its activity ([[dormancy]], [[DNA remanence]]) ([[Prosser2015_Dispersingmisconceptions]])

eDNA persistence duration is being investigated, with varying estimates for plants (50 years in [[Yoccoz2012_DNAsoil]], >60 years in [[Foucher2020_Persistenceenvironmental]])

eDNA techniques allowed for rapid calculation of species richness for arthropods in forests [[Jabot2025_Usemassive]]

#todo/question could we use eDNA databases only at the OTU level (with BINs in the BOLD database) to calculate richness values (at regional scale rather than at local scale - ie gamme vs alpha diversity).

eDNA datasets were used in [[deep species distribution model]]s to derive soil organisms abundance maps using[[soil food web#soil multi-trophic classification]] ([[Calderon-Sanou2022_betterunderstanding]] and [[Cerna2025_Limitspromises]]).

#todo/read [[Orgiazzi2015_Soilbiodiversity]]

#todo/question 
creation of eDNA maps from eDNA samples for species distribution models (SDM)
eDNA data are still sparse and are highly local (points)
SDM require rasters and there is a need to convert eDNA data into rasters by, i suggest, creating buffers around sampled points
deep learning methods could be used to generate eDNA maps based on environmental conditions ? Is it not training by masking ?
## Handles
[[environmental DNA]]
[[species richness]]
[[DNA metabarcoding]]
[[soil biodiversity]]
[[operational taxonomic unit]]